plugins/modules/helm.py

```diff
@@ -98,10 +98,12 @@
   kube_context:
     description:
       - Helm option to specify which kubeconfig context to use.
+      - If the value is not specified in the task, the value of environment variable C(K8S_AUTH_CONTEXT) will be used instead.
```

Comment:
I'm jusst now realizing... it seems these two values are inconsistent with the same values for the other k8s modules, e.g. https://github.com/ansible-collections/community.kubernetes/blob/98b72a4b2dacf447eab96dd6cbf09aa65825554d/plugins/doc_fragments/k8s_auth_options.py#L24-L34

Maybe we should file a follow-up issue to deprecate these parameters and use consistent parameters across all the modules?

Reply:
+1

Reply:
I will work on follow-up PR.